locale/it/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: Consigliata
   changelog: Changelog
   api: Documentazione API
-  version-schedule-prompt: Dai un'occhiata alla
-  version-schedule-prompt-link-text: tabella di marcia LTS
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-it  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?